.github/workflows/playwright.yml

```diff
@@ -4,6 +4,13 @@ on:
     branches: [ main, staging ]
   pull_request:
     branches: [ main, staging ]
+  workflow_run:
```

Comment:
I think that the trigger should be a new release

name: Post Release Workflow

on:
  release:
    types: [published]

jobs:
  post-release:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout code
        uses: actions/checkout@v2

      - name: Run post-release tasks
        run: |
          echo "Release published!"
          # Add your post-release tasks here